UI/Selenium/Steps/ManageWorkAllocationSteps.cs

```diff
@@ -16,10 +16,11 @@ public class ManageWorkAllocationSteps : ObjectFactory
     private readonly ScenarioContext _scenarioContext;
     SelectYourHearingListSteps selectYourHearingListSteps;
```

Comment:
Did you intentionally leave the visibility modifier out for this field?

Reply:
To fix warning

Reply:
I suspect it should be because the field should be 
```suggestion
   private readonly SelectYourHearingListSteps _selectYourHearingListSteps;
```

Reply:
i have apply changes